src/base/static/state/ducks/map.js

```diff
@@ -112,14 +147,42 @@ export default function reducer(state = INITIAL_STATE, action) {
           ...action.payload,
         },
       };
+    case HIDE_OLD_BASEMAP:
+      if (state.visibleBasemapId) {
+        return {
+          ...state,
+          visibleBasemapId: null,
+          layerStatuses: {
+            ...state.layerStatuses,
+            [state.visibleBasemapId]: {
+              ...state.layerStatuses[state.visibleBasemapId],
+              isVisible: false,
+            },
+          },
+        };
+      }
+      return state;
+    case SHOW_BASEMAP:
+      return {
+        ...state,
+        visibleBasemapId: action.payload,
+        layerStatuses: {
+          ...state.layerStatuses,
+          [action.payload]: {
+            ...state.layerStatuses[action.payload],
+            isVisible: true,
+            status: "loading",
+          },
+        },
+      };
```

Comment:
That makes sense. We'll need a separate `initBasemap` action I think, because `state.visibleBasemapId` starts out as `undefined`.

`redux-multi` is still used in the `initLayers` call though, but we could remove it by iterating over the layers and calling separate `initLayer` actions.

Reply:
So in that case we would still use `redux-multi`?

Reply:
No, I don't think we need `redux-multi` :+1: